src/compressed_tensors/compressors/quantized_compressors/base.py

```diff
@@ -82,19 +82,42 @@ def compress(
         """
         compressed_dict = {}
         weight_suffix = ".weight"
+        input_zp_suffix = ".input_zero_point"
+        weight_zp_suffix = ".weight_zero_point"
         _LOGGER.debug(
             f"Compressing model with {len(model_state)} parameterized layers..."
         )
 
         for name, value in tqdm(model_state.items(), desc="Quantized Compression"):
+            # check if the parameter we're compressing is the weight zp
+            # or the input zp
+            weight_zp = name.endswith(weight_zp_suffix)
+            input_zp = name.endswith(input_zp_suffix)
+
+            # if we're compressing the weight zp, fetch weight quant args
```

Comment:
Could we update this comment, since we're not actually compressing the zp's? This is only done to figure out if we should save zp's or not